backpack-common/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -136,7 +136,7 @@ data class CalendarParams(
 data class CellInfo(
     val disabled: Boolean = false,
     val status: CellStatus? = null,
-    val label: String? = null,
+    val label: CellLabel? = null,
```

Comment:
just confirming, is this an intentional breaking change? just to set the correct labels and version backpack accordingly

Reply:
I don't think we should make this a breaking change. Even if we only use it in one place in the Skyscanner app and wouldn't experience a regression ourselves, we need to keep in mind that this is an open sourced library and it should be kept backwards compatible if there isn't a good case for changing this type.

Reply:
Based on the current specifications, we can have both the label and the icon in the same calendar configuration. However, I agree that we should avoid making this change disruptive. I will revert the label and add the icon as a new variable, as suggested below by @andra-georgescu.

Reply:
i've a different view from Andra's on this one, depending on how the component should work, having the 2 properties (a label and an icon prop) could give the impression that you can have the 2 at the same time, wich doest seem to be the case. ie, i could set the label to `$50` and the icon to `.search`, but when I run the app i'd only see one of the 2, giving me unexpected behaviour.

if only one of the 2 needs to be shown at a given time, we need to make the API of the component enforce the correct behaviour, or else it just wont compile. I believe a sealed class is best in this case with associated values (as in, an enum with values) so it can either be a label or an icon, but not both 😬

Reply:
(or have 2 constructors for the data class, that'd also work)